### PR TITLE
hide right panel when input is visible

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes [124](https://github.com/pxblue/angular-design-patterns/issues/124) .

#### Changes proposed in this Pull Request:
- Hide right panel when first screen is visible.

### CODE REVIEW

**Comment by emclaug2** [comment]
This fixes the issue but now the 'Add another device' screen doesn't transition off screen, it just disappears immediately.

**Comment by jeffvg** [comment]
I didn't notice any change in page animations
https://user-images.githubusercontent.com/37150565/132711506-a0f6a3b0-d9c6-4f01-8748-2b4c9ea41061.mov

**Comment by emclaug2** [comment]
@jeffvg in your gif, whenever the "Add a Device" button is clicked, the page content should exit/slide to the right.  Instead it disappears before the input field screen slides in from the left.

**Comment by jeffvg** [comment]
> @jeffvg in your gif, whenever the "Add a Device" button is clicked, the page content should exit/slide to the right. Instead it disappears before the input field screen slides in from the left.

ah ha, yes I see now

**Review by jeffvg — APPROVED** [review]
looks good

**Comment by jeffvg** [comment]
can we sort this new section with this PR

image

**Comment by huayunh** [comment]
(When I did my design I didn't follow any particular order for these patterns, just whichever one comes up to my mind first, so feel free to adjust it to alphabetical, or the order that they are introduced on docit)

**Comment by emclaug2** [comment]
> (When I did my design I didn't follow any particular order for these patterns, just whichever one comes up to my mind first, so feel free to adjust it to alphabetical, or the order that they are introduced on docit)

This is what I did on the react side.

File: src/app/pages/forms-and-validation/verify-on-submit/verify-on-submit.component.html [hunk 1]
```diff
@@ -54,7 +54,13 @@ <h2>Find Device</h2>
         <section class="add-new-device">
             <pxb-empty-state title="Success" description="Device “123” has been added to your repository.">
                 <mat-icon pxb-empty-icon>check_circle</mat-icon>
-                <button mat-raised-button color="primary" pxb-actions (click)="addAnotherDevice()">
+                <button
+                    [attr.tabindex]="isLeftPaneVisible ? -1 : null"
```
Comment (emclaug2) [inline src/app/pages/forms-and-validation/verify-on-submit/verify-on-submit.component.html:58]: Very nice fix.

**Review by emclaug2 — APPROVED** [review]

**Comment by ektasawant** [comment]
> (When I did my design I didn't follow any particular order for these patterns, just whichever one comes up to my mind first, so feel free to adjust it to alphabetical, or the order that they are introduced on docit)

Fixed please review

**Pull request merged**